src/ACadSharp/Objects/CadDictionary.cs

```diff
@@ -283,7 +283,14 @@ public bool ContainsKey(string key)
 		/// <returns>true if the element is successfully removed; otherwise, false.</returns>
 		public bool Remove(string key, out NonGraphicalObject item)
 		{
-			if (this._entries.Remove(key, out item))
```

Comment:
Net framework does not support this method either but there is an implementation in the [CSUtilities](https://github.com/DomCR/CSUtilities).